demo/mock-bank/integration_test.sh

```
@@ -0,0 +1,242 @@
+if [[ -z "${api_key}" ]]; then
+  echo "ERROR: CORDUM_API_KEY must be set (usually from .env)." >&2
+  exit 2
+fi
+
+# Keep cordumctl on the same local TLS gateway as the raw HTTP calls.
+# Otherwise `pack install` falls back to http://localhost:8081 even when the
+# script is polling https://127.0.0.1:8081 via CORDUM_API_BASE.
+export CORDUM_GATEWAY="${CORDUM_GATEWAY:-${api_base}}"
+if [[ -z "${CORDUM_TLS_CA:-}" && -z "${CORDUM_TLS_INSECURE:-}" ]]; then
+  case "${CORDUM_GATEWAY}" in
+    https://127.0.0.1:*|https://localhost:*|https://[::1]:*)
```

Shellcheck (0.11.0)

[info] Ranges can only match single chars (mentioned due to duplicates).

(SC2102)